storage_broker/src/bin/storage_broker.rs

```diff
@@ -674,12 +693,54 @@ async fn main() -> Result<(), Box<dyn std::error::Error>> {
     };
     let storage_broker_server = BrokerServiceServer::new(storage_broker_impl);
 
+    let http_listener = match &args.listen_addr {
+        Some(addr) => {
+            info!("listening HTTP on {}", addr);
+            Some(TcpListener::bind(addr).await?)
+        }
+        None if args.listen_https_addr.is_none() => {
+            info!("listening HTTP on default address {}", DEFAULT_LISTEN_ADDR);
+            Some(TcpListener::bind(DEFAULT_LISTEN_ADDR).await?)
+        }
```

Comment:
When do we ever need to support running without either of the two listen flags?

`neon_local init` launches broker with the flag, so I think it doesn't need this default behavior.

In any way, worth a comment

```suggestion
        /// If neither listen_addr nor listen_https_addr is specified,
        /// the default behavior is to serve HTTP. This is necessary
        /// because ...
        None if args.listen_https_addr.is_none() => {
            info!("listening HTTP on default address {}", DEFAULT_LISTEN_ADDR);
            Some(TcpListener::bind(DEFAULT_LISTEN_ADDR).await?)
        }
```

Reply:
I just tried to save old behavior (use DEFAULT_LISTEN_ADDR if not specified). But it's not needed actually. I removed it